apps/extension/src/components/ui/dropdown-menu.tsx

```
@@ -0,0 +1,257 @@
+'use client';
+
+import * as React from 'react';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)